reddit/plugin_web.go

```diff
@@ -136,10 +136,18 @@ func HandleNew(w http.ResponseWriter, r *http.Request) interface{} {
 		return templateData.AddAlerts(web.ErrorAlert(fmt.Sprintf("Max %d feeds allowed (or %d for premium servers)", GuildMaxFeedsNormal, GuildMaxFeedsPremium)))
 	}
 
+	subreddit := strings.TrimSpace(newElem.Subreddit)
+	subreddit = strings.ToLower(subreddit)
+	subreddit = strings.TrimPrefix(subreddit, "/")
+
+	for ; strings.HasPrefix(subreddit, "r/"); {
```

Comment:
How about splitting at `/` and only taking the last string?

Reply:
To be honest, I think just `subreddit = strings.TrimPrefix(subreddit, "r/")` is enough--no loop. I don't see a real reason to support `r/r/foo`, which is clearly a typo, whereas accepting `r/foo` is a clear improvement.

Reply:
> To be honest, I think just `subreddit = strings.TrimPrefix(subreddit, "r/")` is enough--no loop. I don't see a real reason to support `r/r/foo`, which is clearly a typo, whereas accepting `r/foo` is a clear improvement.

I agree with this.

Reply:
Yeah, I didn't love the idea either, so I'll drop the `for` loop from the behaviour.